parsing/pprintast.ml

```diff
@@ -1535,7 +1535,7 @@ and type_def_list ctxt f (rf, exported, l) =
       else if exported then " ="
       else " :="
     in
-    pp f "@[<2>%s %a%a%a%s%a@]%a" kwd
+    pp f "@[<hov>@[<2>%s %a%a%a%s%a@]@ %a@]" kwd
```

Comment:
Opening two boxes in the same location is often an antipattern. In particular, an increase of indentation of `2` for the attribute seems fine to me? However, you can also cancel the indentation rule with `@;<1 -2>` without opening a second box.

Reply:
It looks like, I managed to make desired changed without opening an extra box.